orderbook/openapi.yml

```diff
@@ -153,8 +153,34 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/FeeInformation"
-        400:
+        404:
           description: sellToken not existent
+  /api/v1/trades/:
+    get:
+      summary: Get existing Trades.
+      description: |
+        By default all trades are returned.
+        Queries can be refined by specifiying owner or order_uid.
+      parameters:
```

Comment:
I think the pagination parameters are missing. 

there will be many trades, so we will need to define some kind of paging params

Reply:
We don't yet have it for orders either. Not sure if we should add it here or later.

Reply:
I think this is something we know we'll need, so I don't see why not to add it.
We cannot expect to return all orders or all trades given a filter criteria of:
- owner
- appId
- dates
- ...
there will be too many results

Reply:
Yes it will be included ASAP, but not as per of this PR.

Reply:
okok 👌